netmanager/src/views/components/Map/Map.js

```diff
@@ -6,7 +6,7 @@ import Select from 'react-select';
 
 import { Map, FeatureGroup, LayerGroup, TileLayer, Marker, Popup } from 'react-leaflet';
 import { EditControl } from 'react-leaflet-draw';
-import axios from 'axios';
+import createAxiosInstance from '../../apis/axiosConfig';
```

Comment:
Hi @OchiengPaul442 
Thanks again!
Just wondering, do we have a way of using simple paths to minimise mistakes?

1. You could try using `Webpack Module Aliases`?
2. Or `Babel Plugin Module Resolver`?
3. Or `Create React App (CRA)`?
4. Or` ESLint "import/no-unresolved" Rule`?
5. Or `Visual Studio Code Path Intellisense`?
6. etc?